### PR TITLE
Add jQuery-QueryBuilder

### PR DESCRIPTION
**Pull request by mistic100**
http://mistic100.github.io/jQuery-QueryBuilder

### CODE REVIEW

File: files/jquery.query-builder/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
+  "name": "jQuery-QueryBuilder",
+  "repo": "mistic100/jQuery-QueryBuilder",
+  "files": {
+    "basePath": "dist/",
+    "include": ["**/*"]
```
Comment (jimaek) [inline files/jquery.query-builder/update.json]: I think this will work better
`./js/*.min.*`,`./css/*.min.*`, `./i18n/*`

Comment (mistic100) [inline files/jquery.query-builder/update.json]: What does "I think" means ? is this a requirement for acceptance or a technical requirement ?

Comment (jimaek) [inline files/jquery.query-builder/update.json]: Its a soft requirement. Unless you can prove you need the excluded files you should use the one I provided.

Comment (mistic100) [inline files/jquery.query-builder/update.json]: fair enough

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by mistic100** [comment]
That's nice automation ^^

**Pull request closed without merging**